File: GoblinVisual/GoblinVisual/GoblinVisual/Requetes/Cash.cs
/** @author Max
*/

using Requete


abstract class Cash : Requete {

	public Cash (int somme) {
		somme = somme;
	}

	protected int somme;

	public getSomme() {
		return somme;
	}


	public override bool doitTraiter(Receptionniste r) {
		return true;
	}
	
	public override bool doitTraiter(Tresorier t) {
		return true;
	}

	public override bool doitTraiter(Tamponneur t) {
		return true;
	}
}

File: Java/Requetes/Cash.java
/** @author Max
*/



public abstract class Cash extends Requete {

	
	protected int somme;

	public Cash(int somme) {
		somme = somme;
	}
		

	public int getSomme() {
		return somme;
	}

	@Override
	public boolean doitTraiter(Receptionniste r) {
		return true;
	}

	@Override
	public boolean doitTraiter(Tresorier t) {
		return true;
	}

	@Override
	public boolean doitTraiter(Tamponeur t) {
		return true;
	}
	
}